x-pack/plugins/maps/public/shared/layers/sources/kibana_tilemap_source/create_source_editor.js

```diff
@@ -27,7 +27,7 @@ export class CreateSourceEditor extends Component {
 
   _loadUrl = async () => {
     const tilemap = await getKibanaTileMap();
-    if (this._isMounted) {
+    if (this._isMounted && tilemap.url) {
```

Comment:
I think the default meta needs to be updated based on this to return an object instead of an array

```
export async function getKibanaTileMap() {
  const dataSource = await getDataSources();
  return _.get(dataSource, 'kibana.tilemap', []);
}
```